Stripe/STPPaymentMethod.m

```diff
@@ -148,10 +153,20 @@ - (NSString *)label {
             }
         case STPPaymentMethodTypeiDEAL:
             return @"iDEAL";
+        case STPPaymentMethodTypeFPX:
+            if (self.fpx != nil) {
+                return STPStringFromFPXBankBrand(STPFPXBankBrandFromIdentifier(self.fpx.bankIdentifierCode));
+            } else {
+                return @"FPX";
+            }
         case STPPaymentMethodTypeCardPresent:
         case STPPaymentMethodTypeUnknown:
             return STPLocalizedString(@"Unknown", @"Default missing source type label");
     }
 }
 
+- (BOOL)reusable {
+    return (self.type == STPPaymentMethodTypeCard);
```

Comment:
is iDEAL not reusable and we've just not run into issues with it because it's not in Standard Integration?

Reply:
Yeah, this is the first non-reusable PaymentMethod we've tried to adopt in Standard Integration. I believe everyone is still using iDEAL as a Source, not a PaymentMethod.